chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -228,6 +228,18 @@ void ContentSettingsRegistry::BraveInit() {
            ContentSettingsInfo::INHERIT_IF_LESS_PERMISSIVE,
            ContentSettingsInfo::EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS);
 
+  // Register AI chat permission default value as Ask.
+  Register(ContentSettingsType::BRAVE_AI_CHAT, "brave_ai_chat",
+           CONTENT_SETTING_ASK, WebsiteSettingsInfo::UNSYNCABLE,
+           /*allowlisted_schemes=*/{},
+           /*valid_settings=*/
+           {CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK, CONTENT_SETTING_ASK},
+           WebsiteSettingsInfo::TOP_ORIGIN_ONLY_SCOPE,
+           WebsiteSettingsRegistry::DESKTOP |
+               WebsiteSettingsRegistry::PLATFORM_ANDROID,
```

Comment:
Isn't this feature Desktop-only?

Reply:
removed

Reply:
As mentioned in DM, added back to avoid unnecessary is_android guards.